packages/smithy-aws-event-streams/pyproject.toml

```diff
@@ -1,7 +1,7 @@
 [project]
-name = "aws-event-stream"
+name = "smithy-aws-event-stream"
```

Comment:
I see some other smithy packages we have use snake case. Does this get treaded the same as `smithy_aws_event_stream`? We should make them consistent.

Reply:
It looks like none of the projects are consistent. I'll do a follow up PR to normalize everything.